contracts/smartwallet/SmartWallet.sol

```diff
@@ -199,7 +200,7 @@ contract SmartWallet is IForwarder {
         bytes32 suffixData,
         ForwardRequest memory req,
         bytes memory sig
-    ) private view {
+    ) internal view {
```

Comment:
this is my main concern, the private methods are not copied from the father and you cannot and cannot be overriden, we could either create the same method with a different name or keep it with internal, or maybe Im missing something.